docs/STABLE.md

```diff
@@ -47,7 +49,7 @@ Verify the tag SHA hashes look correct:
 $ git show-ref --tags
 ```
 
-Push the new tag to remote (origin) with `-f` (force) flag.
+Push the new tag to your fork (origin) with `-f` (force) flag.
```

Comment:
This section seems not useful. It assumes my fork has a stable tag. But forking in the github UI only copies the main branch (by default) to the fork. My fork has no tags since I never pushed any to my fork.

Reply:
@markstur any thoughts on that? Happy to remove this section but not sure if you had a specific intention for having it in here I'm not aware of

Reply:
In this case "origin" is upstream.  So it has tags.   I guess this needs to be clarified for folks that use a fork as origin.

I have only been tagging directly on the upstream repo.  Never tagged my fork.  So there are no instructions here to tag a fork and push from there somehow.